pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -270,7 +270,6 @@ type PodSecurityPolicyTemplateProjectBinding struct {
 }
 
 // +genclient
-// +kubebuilder:skipversion
```

Comment:
On the ProjectRoleTemplateBinding struct, can you update the JSON tag for `RoleTemplateName` and `ProjectName` The generate CRD does not have them as required because the `,omitempty` tag is present.